hoot-core/src/main/cpp/hoot/core/scoring/GraphComparator.cpp

```diff
@@ -414,7 +414,7 @@ void GraphComparator::_exportGraphImage(OsmMapPtr map, DirectedGraph& /*graph*/,
     }
     else
     {
-      int v = (cost / maxCost) * 255.0;
+      double v = (cost / maxCost) * 255.0;
       c.setRgb(255, v, 0);
```

Comment:
If you change `v` to `double` then the following line needs to cast `v` to `int`.  My preference would be to keep `v` an `int` and do the cast on assignment:
```
int v = std::static_cast<int>((cost / maxCost) * 255.0);
```